xml/src/main/java/com/baeldung/xmlhtml/pojo/jaxb/html/elements/CustomElement.java

```diff
@@ -0,0 +1,25 @@
+package com.baeldung.xmlhtml.pojo.jaxb.html.elements;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlValue;
+
+/**
+ * @author Adam I. Gerard
+ */
+
+@XmlAccessorType(XmlAccessType.PROPERTY)
```

Comment:
Why do we use XmlAccessType.PROPERTY on some classes, but PUBLIC_MEMBER on others? Let's make them all the same.